patches/minecraft/net/minecraft/entity/Entity.java.patch

```diff
@@ -9,7 +9,15 @@
  {
      private static final Logger field_184243_a = LogManager.getLogger();
      private static final List<ItemStack> field_190535_b = Collections.<ItemStack>emptyList();
-@@ -197,7 +197,7 @@
+@@ -175,6 +175,7 @@
+     private boolean field_184237_aG;
+     private double[] field_191505_aI;
+     private long field_191506_aJ;
++    public boolean updateBlocked;//Setting this to true will prevent the world from calling onUpdate for this entity. 
```

Comment:
can you make this a javadoc comment? Then we'll see it on mouseover everwhere this field is used. Or would this make the patch too big?